types/chrome/chrome-cast/index.d.ts

```diff
@@ -373,7 +373,7 @@ declare namespace chrome.cast {
          */
         sendMessage(
             namespace: string,
-            message: string,
+            message: string | Object,
```

Comment:
```suggestion
            message: string | object,
```
If you use `Object` then pretty much anything but `null` and `undefined` are assignable to `message`.